packages/flutter/lib/src/widgets/form.dart

```diff
@@ -114,11 +119,6 @@ class Form extends StatefulWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
-  /// If true, form fields will validate and update their error text
-  /// immediately after every change. Otherwise, you must call
-  /// [FormState.validate] to validate.
-  final bool autovalidate;
```

Comment:
Turns out this is a breaking change. The public field is removed, not deprecated.

This affects (at least) [flutter_form_builder](https://pub.dev/packages/flutter_form_builder).
https://github.com/danvick/flutter_form_builder/issues/465

Ideally, should (with appropriate deprecated tag) put something like
```dart
bool get autovalidate => autovalidateMode != AutovalidateMode.disabled;
```

Reply:
@pedromassango oops did we miss this one?

Reply:
Hmm. Just checked `form.dart` file and I don't see any reference for `final bool autovalidate`. We did removed this intentionally in favour of AutovalidateMode property.

Maybe this is referencing a old commit file?!

Reply:
> bool get autovalidate => autovalidateMode != AutovalidateMode.disabled;

This is interesting and would prevent the breaking changes 🤔 
@chunhtai do you think this we should put it back?

Reply:
@skyeskie
What are the use cases of using this Widget's property?

Reply:
I think we should put this one back, but we should do
```
autovalidateMode = autovalidateMode ?? autovalidate ? AutovalidateMode.always : AutovalidateMode.disabled
```
Same as what we do for other formfield

Reply:
Alright, I will send a PR to put it back ASAP

Reply:
`FormField` has the exact same issue (below, 363-368)

@pedromassango
I personally don't use this, but just ran across when `flutter_form_builder` refused to compile, specifically when it tries to access `autovalidate` on `FormField` in its `FormBuilderCustomField` class [here](https://github.com/danvick/flutter_form_builder/blob/2d5f61b3863b058b9b6213aac25f6b62b65bd44e/lib/src/form_builder_custom_field.dart#L96).

Reply:
The PR is: https://github.com/flutter/flutter/pull/66267

@chunhtai I think we should add it back on formField widget too, right?